test/hotspot/jtreg/compiler/c2/aarch64/libTestSVEWithJNI.c

```diff
@@ -23,7 +23,7 @@
 *
 */
 
-#ifdef __aarch64__
+#if defined(__aarch64__) && defined(LINUX)
```

Comment:
I have filled separate bug for this https://bugs.openjdk.java.net/browse/JDK-8254670 as it's not supposed to be a  part of jep388/391

Reply:
Let's keep it here until the fix lands in master and we can merge it back into this branch.